app/config.matka.js

```diff
@@ -0,0 +1,450 @@
+const CONFIG = process.env.CONFIG || 'default';
```

Comment:
Only need values that are different from the default config. See eg. HSL's config